.pre-commit-config.yaml

```diff
@@ -36,20 +36,29 @@ repos:
       # known problem with hooks in subdirectories when pass_filenames option
       # is set to true
       # See: https://github.com/pre-commit/pre-commit/issues/1417
-      - id: prettier
-        name: Prettier
+      - id: prettier-app
+        name: Prettier App
         # NOTE: This hook currently does not work on Windows due to "yarn" not being an executable and win32api.CreateProcess
         # turning `subprocess.run(["yarn", "prettier", "--write"])` into a call to `yarn.exe prettier --write` which does not exist
-        entry: ./scripts/run_in_subdirectory.py frontend/ yarn prettier --write
-        files: ^frontend/.*\.(js|jsx|ts|tsx)$
+        entry: ./scripts/run_in_subdirectory.py frontend/app yarn run format
+        files: ^frontend/app/.*\.(js|jsx|ts|tsx)$
+        exclude: /vendor/
+        language: node
+        pass_filenames: true
+      - id: prettier-lib
+        name: Prettier Lib
+        # NOTE: This hook currently does not work on Windows due to "yarn" not being an executable and win32api.CreateProcess
+        # turning `subprocess.run(["yarn", "prettier", "--write"])` into a call to `yarn.exe prettier --write` which does not exist
+        entry: ./scripts/run_in_subdirectory.py frontend/lib yarn run format
+        files: ^frontend/lib/.*\.(js|jsx|ts|tsx)$
         exclude: /vendor/
         language: node
         pass_filenames: true
       - id: prettier-yaml
         name: Prettier-yaml
         # NOTE: This hook currently does not work on Windows due to "yarn" not being an executable and win32api.CreateProcess
         # turning `subprocess.run(["yarn", "prettier", "--write"])` into a call to `yarn.exe prettier --write` which does not exist
-        entry: ./scripts/run_in_subdirectory.py frontend yarn prettier ../.github --write
+        entry: ./scripts/run_in_subdirectory.py frontend/app yarn prettier ../../.github --write
```

Comment:
Why is this change here needed?

Reply:
Yarn is strict on not using scripts in folders that don't have them in dependencies. The frontend directory does not have prettier, but both lib and app do. Since we are applying these on the .github folder (in neither, I just chose one to leverage its prettier).

Future work would be to break out the eslint/prettier logic into another package for reuse.